README.md

```diff
@@ -92,9 +92,9 @@ a new release package. Release packages are identified by 3 elements:
 - The application name
 - The release version in SemVer fashion (by default `0.1.0`)
 
-This command has 2 options:
+This command has 1 optional option:
 - `--force` or `-f`: This option will force an overwritting of the local and remote files of a given project and release
-- `--no-pack` or `-p`: With this option enabled, the command won't run internally the command `rebar3 grisp pack`
```

Comment:
Maybe replace it but the refresh option: `--refresh` `-r`: Force software package building even if it already exists